test/AspNet.Security.OAuth.Providers.Tests/GitCode/bundle.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "https://raw.githubusercontent.com/justeat/httpclient-interception/master/src/HttpClientInterception/Bundles/http-request-bundle-schema.json",
+  "items": [
+    {
+      "uri": "https://gitcode.com/oauth/token",
+      "method": "POST",
+      "contentFormat": "json",
+      "contentJson": {
+        "access_token": "secret-access-token",
+        "expires_in": 1296000,
+        "refresh_token": "secret-refresh-token",
+        "scope": "all_user all_key all_groups all_projects all_pr all_issue all_note all_hook all_repository",
+        "created_at": "2024-04-20T09:07:59.889Z"
+      }
+    },
+    {
+      "comment": "https://docs.gitcode.com/en/docs/apis/get-api-v-5-user/",
+      "uri": "https://api.gitcode.com/api/v5/user",
+      "contentFormat": "json",
+      "contentJson": {
+        "avatar_url": "https://cdn-img.gitcode.com/fa/be/example.png?time=1694709764757",
+        "followers_url": "https://api.gitcode.com/api/v5users/example/followers",
```

Comment:
?

```suggestion
        "followers_url": "https://api.gitcode.com/api/v5/users/example/followers",
```

Reply:
This is sample data from [the GitCode documentation](https://docs.gitcode.com/en/docs/users/). After comparing it with the actual returned data, it seems that the data they provided is incorrect. I appreciate your reminder.

Reply:
Documentation:
![image](https://github.com/user-attachments/assets/0b3a55da-41fa-4552-a0ea-7d1ca23b91f6)

Actual response:
![8563b2ce2309d6da956fbd37279f5a25](https://github.com/user-attachments/assets/356b3dd6-cf96-48fb-9175-c9705447564f)

Feedback has been submitted to them.